java/src/org/openqa/selenium/netty/server/RequestConverter.java

```diff
@@ -67,6 +67,7 @@ protected void channelRead0(ChannelHandlerContext ctx, HttpObject msg) throws Ex
       io.netty.handler.codec.http.HttpRequest nettyRequest =
           (io.netty.handler.codec.http.HttpRequest) msg;
 
+      request = createRequest(ctx, nettyRequest);
       if (HttpUtil.is100ContinueExpected(nettyRequest)) {
         ctx.write(new HttpResponse().setStatus(100));
         return;
```

Comment:
This `if` block should be moved below the `if (request == null) {` block to ensure we do set the request in case of a protocol upgrade.

Reply:
Please re-review, resolved comment